README.md

```diff
@@ -17,7 +17,7 @@ Here are the goals of `level-js`:
 - Pass the full `abstract-leveldown` test suite
 - Support [Buffer](https://nodejs.org/api/buffer.html) values (in all target environments)
 - Support all key types of IndexedDB Second Edition, including binary keys (depends on environment)
-- Support all value types of the structured clone algorithm (depends on environment)
+- Support all value types of the structured clone algorithm (depends on environment) except for `null` and `undefined`
```

Comment:
Should we mention something about writing them as empty strings if they are used as values?

Reply:
Yes. We can do that as part of #123

Reply:
(which is a bigger effort)